docs/admin/access.rst

```diff
@@ -1,333 +1,433 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    Can manage glossary (add or remove entries, or upload).
+A user can access unrestricted components once able to access the components'
+project (and will have all the permissions he or she was granted for the
+project). With :ref:`component-restricted` turned on, access to the component
+requires explicit permissions for the component (or a component list it is in).
 
-.. describe:: Memory
+.. _perm-check:
 
-    Can manage translation memory.
+Scope of groups
+^^^^^^^^^^^^^^^
 
-.. describe:: Screenshots
+The scope of the permission assigned by the roles in the groups are applied by
+the following rules:
 
-    Can manage screenshots (add or remove them, and associate them to source
-    strings).
+- If the group specifies any :guilabel:`Component list` all the permissions given to
+  the members of that group are granted for all the components in the component
+  lists attached to the group and an access with no additional permissions is
+  granted for all the projects these components are in. :guilabel:`Components`
+  and :guilabel:`Projects` are ignored.
 
-.. describe:: Review
+- If the group specifies any :guilabel:`Components` all the permissions given to
+  the members of that group are granted for all the components attached to the
+  group and an access with no additional permissions is granted for all the
+  projects these components are in. :guilabel:`Projects` are ignored.
 
-    Can approve translations during review.
+- Otherwise if the group specifies any :guilabel:`Projects` either by directly
+  listing them or by :guilabel:`Selected projects` set to a value like `All
+  public`, all the permissions are applied to all the projects, which
+  effectively grants the same permissions to access all the projects'
+  :ref:`unrestricted components <component-restricted>`.
 
-.. describe:: VCS
+- The restrictions imposed by group's :guilabel:`Languages` are applied later,
+  when the access to perform certain action is checked, and it only affects just
+  some translation-related permissions.
 
-    Can manage VCS and access the exported repository.
+.. hint::
 
-.. describe:: Administration
+   Use :guilabel:`Language selection` or :guilabel:`Project selection`
+   to automate inclusion of all languages or projects.
 
-    Has all permissions available in the project.
+**Example:**
 
-.. describe:: Billing
+  Let's say there is a project ``foo`` with two component: ``foo/bar`` and
+  ``foo/baz`` and the next group:
 
-    Can access billing info (see :ref:`billing`).
+  .. list-table:: Group `Spanish Admin-Reviewers`
+         :stub-columns: 1
 
+         * - Roles
+           - `Review Strings`, `Manage repository`
+         * - Components
+           - foo/bar
+         * - Languages
+           - `Spanish`
 
-.. _custom-acl:
+..
 
-Custom access control
----------------------
+  Members of that group will have next permissions (assuming the default role settings):
```

Comment:
```suggestion
  Members of that group will have the next permissions (assuming the default role settings):
```
The next what? The next line, secondary…?

Reply:
following is right

Reply:
Changed to "following"

Reply:
it’s already changed in the file.